### DISCUSSION TITLE
Jump to previous selcted nodes

### DISCUSSION THREAD

**Discussion by fgionghi** [freeplane/freeplane#2135] [Help]
Use case: I am working on X and suddently it came to my mind something I should add to project Y. So I use https://github.com/lilive/Freeplane-Jumper to find the right node in project Y and paste or add what I have to add. Awesome. Now I want to go back to the node that it was previously selected. My maps are very big and at the moment I have to reach the mouse and find the previous node I was working on.
How can I jump back to the previously selected node?

Thank you

**Comment by euu2021** [comment]
3 ideas:

- Use `Navigate->Go backward`

- or try [UtilityPanels add-on](https://github.com/euu2021/Freeplane_UtilityPanels/tree/main)

- [my favorite] or, when you want to navigate to project y, create a new map view (`View->New map view`), do your thing there, then close the map view. The first map view will be at the point you were working before creating the new map view. Related to that, see https://sourceforge.net/p/freeplane/discussion/758437/thread/986addc0ee/#b9bb

**Reply by fgionghi** [reply]
That is awesome!
There is a place where I can find all the addons? Do you have few more addons you use daily or that I should try?
Thank you

**Reply by euu2021** [reply]
> There is a place where I can find all the addons? Do you have few more addons you use daily or that I should try?

Take a look at this page:
https://docs.freeplane.org/scripting/Finding_useful_scripts.html
